src/NuGet.Core/NuGet.PackageManagement/IDE/PackageRestoreManager.cs

```diff
@@ -57,6 +70,41 @@ public virtual async Task RaisePackagesMissingEventForSolutionAsync(string solut
             PackagesMissingStatusChanged?.Invoke(this, new PackagesMissingStatusEventArgs(missing));
         }
 
+        public virtual async Task<bool> GetMissingAssetsFileStatusAsync(string projectId)
```

Comment:
Is this called anywhere?

Reply:
Not in this file, should I try and move this? I only use it in PackageRestoreBar.xaml.cs and the test file, should I move this to the UI and create a new function in the tests? We can avoid a public api this way

Reply:
Yeah, it'd be great if we can avoid the public API.